code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -112,62 +113,9 @@
 		new cash(src)
 	new coin(src)
 
-//////////////////////////////////////
-//			Color Wallets			//
-//////////////////////////////////////
 
-/obj/item/storage/wallet/color
+// Arcade Wallet
+/obj/item/storage/wallet/cheap
 	name = "cheap wallet"
-	desc = "A cheap wallet from the arcade."
+	desc = "A cheap and flimsy wallet from the arcade."
 	storage_slots = 5		//smaller storage than normal wallets
-
-/obj/item/storage/wallet/color/Initialize(mapload)
-	. = ..()
-	if(!item_color)
-		var/color_wallet = pick(subtypesof(/obj/item/storage/wallet/color))
-		new color_wallet(loc)
-		qdel(src)
-		return
-	update_appearance(UPDATE_NAME|UPDATE_DESC|UPDATE_ICON_STATE)
-
-/obj/item/storage/wallet/color/update_desc(updates)
-	. = ..()
-	desc = "A cheap, [item_color] wallet from the arcade."
-
-/obj/item/storage/wallet/color/update_icon_state()
-	if(front_id)
-		switch(front_id.icon_state)
-			if("silver")
-				icon_state = "[item_color]_walletid_silver"
-				return
-			if("gold")
-				icon_state = "[item_color]_walletid_gold"
-				return
-			if("centcom")
-				icon_state = "[item_color]_walletid_centcom"
-				return
-			else
-				icon_state = "[item_color]_walletid"
-				return
-	icon_state = "[item_color]_wallet"
-
-/obj/item/storage/wallet/color/blue
-	item_color = "blue"
-
-/obj/item/storage/wallet/color/red
-	item_color = "red"
-
-/obj/item/storage/wallet/color/yellow
-	item_color = "yellow"
-
-/obj/item/storage/wallet/color/green
-	item_color = "green"
-
-/obj/item/storage/wallet/color/pink
-	item_color = "pink"
-
-/obj/item/storage/waller/color/brown
-	item_color = "brown"
```

Comment:
brown arcade wallets never existed because of `waller`. Thats my favorite ss13 fact. I only found it because I made the same typo as they did 7 years ago